include/nwnx_cplugin.h

```diff
@@ -11,8 +11,9 @@ struct NWNXCPlugin_InitInfo
 {
 	/// Path to this plugin DLL file
 	const char* dll_path;
-	/// Path to the NWNX4 base directory, where nwnx4_controller.exe is located.
-	const char* nwnx_path;
+    /// Path to the NWNX4 user directory, where config files are stored and log files should be
+    /// written.
+	const char* nwnx_user_path;
```

Comment:
We're telling them where the configurations are, but not where the base NWNX4 install is?